docs/README-agent-install.md

```diff
@@ -164,318 +153,303 @@ Virtual service guestshell+ detail
-* NX-OS:
-  * `bash-shell`: Use [http://yum.puppetlabs.com/puppetlabs-release-pc1-cisco-wrlinux-5.noarch.rpm](http://yum.puppetlabs.com/puppetlabs-release-pc1-cisco-wrlinux-5.noarch.rpm)
-  * `guestshell`: Due to issue https://tickets.puppetlabs.com/browse/FACT-1409, please install a `1.3.x` version of the puppet agent RPM inside the `guestshell`.
-    * Latest `1.3.x` version at the time of publishing this module:
-      * [http://yum.puppetlabs.com/el/7/PC1/x86_64/puppet-agent-1.3.6-1.el7.x86_64.rpm](http://yum.puppetlabs.com/el/7/PC1/x86_64/puppet-agent-1.3.6-1.el7.x86_64.rpm)
-    * Once this issue is resolved use [http://yum.puppetlabs.com/puppetlabs-release-pc1-el-7.noarch.rpm](http://yum.puppetlabs.com/puppetlabs-release-pc1-el-7.noarch.rpm)
-  * `open agent container (OAC)`: Use [http://yum.puppetlabs.com/puppetlabs-release-pc1-el-6.noarch.rpm](http://yum.puppetlabs.com/puppetlabs-release-pc1-el-6.noarch.rpm)
+<br>
 
-Using the appropriate RPM for your environment as described above, do:
+* Install the RPM (`$PUPPET_RPM` is the URL from the preceding table)
 
 ~~~bash
 yum install $PUPPET_RPM
 yum install puppet
 ~~~
 
-Update PATH var:
+* Update the PATH variable
 
 ~~~bash
 export PATH=/opt/puppetlabs/puppet/bin:/opt/puppetlabs/puppet/lib:$PATH
 ~~~
 
-#### Install the cisco_node_utils gem
+<br>
+#### *Step 2. Configure* `/etc/puppetlabs/puppet/puppet.conf`
 
-If you wish to make use of the types and providers in this module, you will need to install (and perhaps configure) the [`cisco_node_utils`](https://rubygems.org/gems/cisco_node_utils) Ruby gem. Refer to [README-gem-install.md](README-gem-install.md) for detailed steps.
+Add your Puppet Server name to the configuration file.
 
-#### Edit the Puppet config file:
+~~~bash
+[main]
+  server = mypuppetmaster.mycompany.com
+~~~
+<br>
+
+#### *Step 3. The `cisco_node_utils` Gem*
+
+The [`cisco_node_utils`](https://rubygems.org/gems/cisco_node_utils) ruby gem is a required component of the `ciscopuppet` module. This gem contains platform APIs for interfacing between Cisco CLI and Puppet agent resources. The gem can be automatically installed by Puppet agent by simply using the [`ciscopuppet::install`](https://github.com/cisco/cisco-network-puppet-module/blob/master/examples/demo_all_cisco.pp#L19) helper class, or it can be installed manually.
 
-**/etc/puppetlabs/puppet/puppet.conf**
+##### Automatic Gem Install Using `ciscopuppet::install`
 
-This file can be used to override the default Puppet settings. At a minimum, the following settings should be used:
+* The `ciscopuppet::install` class is defined in the `install.pp` file in the `examples` subdirectory. Copy this file into the `manifests` directory as shown:
 
 ~~~bash
-[main]
-  server = mypuppetmaster.mycompany.com
+cd /etc/puppetlabs/code/environments/production/modules/ciscopuppet/
+cp examples/install.pp  manifests/
+~~~
+
+* Next, update `site.pp` to use the install class
+
+**Example**
+
+~~~
+node default
```

Comment:
Same comments apply here as in: https://github.com/cisco/cisco-network-puppet-module/pull/335/files